ur_controllers/README.md

```diff
@@ -16,3 +16,56 @@ This package contains controllers not being available in the default ROS-Control
 created to support more features offered by the UR robot family. Any of these controllers are
 example implementations for certain features and are intended to be generalized and merged
 into the default ROS-Control controller set at some future point.
+
+## Controller description
+This packages offers a couple of specific controllers that will be explained in the following
+sections.
+### ur_controllers/SpeedScalingStateController
+This controller publishes the readings of the actual execution speed as reported from the robot.
+This is the percentage of the speed slider multiplied with the `target_speed_fraction` that results
```

Comment:
"the speed slider": which speed slider?

And where does `target_speed_fraction` come from? Is that an RTDE field?